gallia-prehistoire.csl

```diff
@@ -241,14 +231,9 @@
         </choose>
         <text macro="issue" prefix=", "/>
       </if>
-      <else-if type="chapter entry-dictionary entry-encyclopedia" match="any">
-        <group delimiter=", ">
-          <text variable="publisher-place" font-style="italic"/>
-        </group>
-      </else-if>
       <else-if type="manuscript thesis report" match="any">
         <group delimiter=", ">
-          <text variable="publisher-place"/>
+          <text variable="locator"/>
```

Comment:
this seems odd - why put the locator here? The locator is the pinpoint assigned to a given citation -- included here, it will never actually be used (this is also the case for the locator in the `vol-pages` macro. Could you say what the goal here is?

Reply:
yes you're right, this is a mistake and a confusion with the pblisher place. I've deleted this condition for thesis/reports and in the vol-pages macro too.